app/assets/stylesheets/application.scss

```diff
@@ -478,4 +478,41 @@ footer ul li a:hover, footer ul li a:focus, footer ul li a:active, footer ul li
 .btn-xl {
   font-size: 11px;
   padding: 15px 45px;
-}
\ No newline at end of file
+}
```

Comment:
There we go! This guy knows CSS!

Reply:
I didn't know he was already full stack?! This is great.